components/cam/src/physics/cam/check_energy.F90

```diff
@@ -841,6 +841,15 @@ subroutine check_energy_fix(state, ptend, nstep, eshflx)
     if (single_column .and. use_replay) then
       heat_glob = heat_glob_scm(1)
     endif
+    
+    ! If single column model we do NOT want to take into
+    !   consideration the dynamics energy fixer.  Since only
+    !   only column of dynamics is active, this data will 
+    !   essentially be garbage. 
```

Comment:
The codes introduced are consistent with the commit description. Just a few typo errors in the comments.  I think you must have meant " In single column model .... Since only one column ...".

Reply:
good catch, fixed and pushed the comment fix